automatic/openssl.light/tools/chocolateyInstall.ps1

```diff
@@ -20,5 +25,7 @@ $packageArgs = @{
 }
 
 Install-ChocolateyInstallPackage @packageArgs
+$binPath = "$($dir)\bin" 
+Install-ChocolateyPath -PathToInstall $binPath 
```

Comment:
👍 
Another note, if you go with the `Get-AppInstallLocation` (which would be preferred), you can most likely just reuse the `softwareName` used in the `$packageArgs` variable.